src/nibetaseries/interfaces/nilearn.py

```diff
@@ -32,27 +32,28 @@ class AtlasConnectivity(NilearnBaseInterface, SimpleInterface):
     def _run_interface(self, runtime):
         from nilearn.input_data import NiftiLabelsMasker
         from nilearn.connectome import ConnectivityMeasure
+        from sklearn.covariance import EmpiricalCovariance
         import numpy as np
         import pandas as pd
         import os
 
         # extract timeseries from every label
-        masker = NiftiLabelsMasker(labels_img=self.inputs.atlas_file, standardize=True,
```

Comment:
also removed standardize and memory options from the masker, may want to keep standardize so that variances are comparable across regions of interest.